src/shared/components/results/ResultsButtons.tsx

```diff
@@ -109,6 +113,7 @@ const ResultsButtons: FC<ResultsButtonsProps> = ({
           title={`Switch to "${
             viewMode === ViewMode.CARD ? 'table' : 'card'
           }" view`}
+          disabled={namespace === Namespace.idmapping || disableCardToggle}
```

Comment:
I'm thinking that having `disableCardToggle` at our disposal means we don't need to additionally check `namespace === Namespace.idmapping`

Reply:
You're right, it's just that as per my comment I thought we would eventually remove `disableCardToggle` so I'd left the `idmapping` check. However now that I'm working on the download I realise we will have to pass an "isIDMapping" flag down so might change that...

Reply:
We have the card view for peptide-search, so again raising the point of:
 - is it worth it to add edge-cases here and there to force only ID mapping to only display table, when it might be easier to just display the cards and have greater consistency across the website?

Reply:
I'm still not convinced cards make sense for a mapping to which is not a namespace such as Ensembl etc... but agree that eventually we could have cards for UniProtkb/UniRef/UniParc (which is why I added the note here https://github.com/ebi-uniprot/uniprot-website/pull/374/files/081ac813a5467323d4c187181c81f4009cfffb86#diff-e631b19484cab2883e9c24ccd391fb48f52a2a72ce4588e82bbe44eef6addd60R37

Reply:
Agree that it isn't an improvement as it doesn't _add_ anything visually, but:
 - it adds consistency
 - it removes code complexity (the big issue for me)
 - it's not less useful than a 2-column table

Should we involve UX?

Reply:
Yes, definitely, but the current aim is to get this functionality out (replicating what is in the current website) as quickly as possible. This is why I'm not adding cards right now.